stetho-repl-js/README.md

```diff
@@ -0,0 +1,223 @@
+# Stetho's Javascript Console
+
+This [Stetho](https://facebook.github.io/stetho) plugin adds a javascript console by embedding Mozilla's [Rhino](https://github.com/mozilla/rhino).
+
+## Set-up
+
+### Download
+Download [the latest JARs](https://github.com/facebook/stetho/releases/latest) or grab via Gradle:
+```groovy
+compile 'com.facebook.stetho:stetho-repl-js:1.1.1'
+```
+or Maven:
+```xml
+<dependency>
+  <groupId>com.facebook.stetho</groupId>
+  <artifactId>stetho-repl-js</artifactId>
+  <version>1.1.1</version>
+</dependency>
+```
+
+Make sure that you depend on the main `stetho` dependency too.
+
+### Putting it together
+The javascript integration is similar to standard Stetho integration.
+The main difference is with the WebKitInspector used.
+There is a simple initialization step which occurs in your `Application` class:
+
+```java
+public class MyApplication extends Application {
+  public void onCreate() {
+    super.onCreate();
+    JavaScriptRuntimeBuilder jsRuntimeBuilder = new JavaScriptRuntimeBuilder(this);
+    Stetho.initialize(
+        Stetho.newInitializerBuilder(this)
+            .enableDumpapp(Stetho.defaultDumperPluginsProvider(this))
+            .enableWebKitInspector(jsRuntimeBuilder.javaScriptInspectorModulesProvider())
```

Comment:
Yes, I'm imagining I'd do this work in a separate PR I was just curious what you thought about Stetho providing this kind of API.  We've got other issues just like the `stetho-repl-js` which just end up looking ugly and hacky when they all get merged into one initialize step.

For instance, folks can configure which directory hosts their databases as well using the same trick that `stetho-repl-js` has to use.